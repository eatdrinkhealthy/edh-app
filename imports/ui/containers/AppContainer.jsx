import React from "react";
import LocationsMap from "../components/LocationsMap";
import Navbar from "../components/Navbar";

const AppContainer = () => (
  <div>
    <Navbar />
    <LocationsMap />
  </div>
);

export default AppContainer;
