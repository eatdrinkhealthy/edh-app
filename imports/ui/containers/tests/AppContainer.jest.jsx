/* eslint-env jest */
/* eslint-disable func-names, prefer-arrow-callback, no-unused-expressions */
/* eslint-disable import/no-extraneous-dependencies */

import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";
import AppContainer from "../AppContainer";

describe("<AppContainer />", function () {
  it("matches render snapshot", function () {
    const tree = renderer.create(<AppContainer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("renders Navbar and LocationsMap components", function () {
    const wrapper = shallow(<AppContainer />);
    expect(wrapper.find("Navbar").length).toBe(1);
    expect(wrapper.find("LocationsMap").length).toBe(1);
  });
});
