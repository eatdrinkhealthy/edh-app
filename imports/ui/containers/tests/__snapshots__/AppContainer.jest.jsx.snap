exports[`<AppContainer /> matches render snapshot 1`] = `
<div>
  <div
    className="nav">
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="toggle-sidebar">
        <div
          className="burger" />
        <div
          className="burger" />
        <div
          className="burger" />
      </div>
    </a>
    <div
      className="nav__title">
      Eat Drink Healthy
    </div>
    <a
      onClick={[Function]}
      style={Object {}}>
      <div
        className="toggle-filter">
        Filter
      </div>
    </a>
  </div>
</div>
`;
